unicodetools/data/ucd/dev/IndicPositionalCategory.txt

```diff
@@ -300,6 +300,9 @@ ABEC          ; Right # Mc       MEETEI MAYEK LUM IYEK
 11F03         ; Right # Mc       KAWI SIGN VISARGA
 11F34..11F35  ; Right # Mc   [2] KAWI VOWEL SIGN AA..KAWI VOWEL SIGN ALTERNATE AA
 11F41         ; Right # Mc       KAWI SIGN KILLER
+16D40..16D42  ; Right # Lm   [3] KIRAT RAI SIGN ANUSVARA..KIRAT RAI SIGN VISARGA
```

Comment:
OK, but the Tai Viet and New Tai Lue characters are not combining marks either.
Are those scripts there only because they have some Visual_Order_Left characters?

Reply:
If there are implied constraints like this, then we should either document them, or have a test for them, or both.

Reply:
Thai, Lao, and scripts *like* Thai and Lao, get the full treatment in InPC. That means that they have full-fledged *letters* (gc=Lo) with InPC=Right assignments, too.
The fundamental first question is whether Kirat Rai needs to be in InSC and InPC at all. It was encoded in a way to explicitly avoid combining marks and pretty much to lay out as simply as possible, as if it were an alphabet. *If* Kirat Rai belongs in InSC and InPC, then all the pieces, formally combining or not, should be assigned in a way similar to Thai, IMO. The proposal didn't provide Indic properties, and I don't recall the SAH asking for them.

Reply:
For what it’s worth, there are three scripts already that are in InSC but not in InPC: Multani, Phags_Pa, and Tai_Le.